WordPress/Classes/Extensions/NSAttributedString+Helpers.swift

```diff
@@ -1,7 +1,7 @@
 import Foundation
 
-
-extension NSAttributedString {
+@objc
```

Comment:
I'm not sure this actually needs to be `@objc`.

Reply:
Both of the messages in this extension were [originally](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Extensions/NSAttributedString%2BHelpers.swift) annotated with `@objc`. I promoted this to the extension for organizational purposes.

Let me know if it's okay to keep it, or if you'd like me to explore removing them both as part of this PR.